src/main/webapp/app/overview/exercise-details/exercise-details-student-actions.component.html

```diff
@@ -174,9 +174,25 @@
             [buttonLoading]="exercise.loading"
             [smallButton]="smallButtons"
             [hideLabelMobile]="false"
-            *ngIf="isOnlineEditorAllowed() && participationStatus() == INITIALIZED && exercise.participations && exercise.participations.length > 0"
+            *ngIf="!isIntelliJ && isOnlineEditorAllowed() && participationStatus() == INITIALIZED && exercise.participations && exercise.participations.length > 0"
             routerLink="/code-editor/{{ exercise.participations[0].id }}"
         ></button>
+        <button
```

Comment:
I think this is too much, especially considered that we are in a very early stage of the plugin's development.
Imo we would be better of to build some new lightweight, intellij wrapper components instead of trying to re-use the old ones here.

Also: Have you considered using Angular elements for this? https://angular.io/guide/elements
They seem like a good fit to me, as the plugin user doesn't really need the whole Artemis web client, just very small portions of it.

Reply:
True!. I would suggest, that I create a new student-ide-actions component. 
About the elements: That's a really cool suggestion. I would like to talk to you about this in the next developer meeting, if it's ok with you?